config/rbac/role.yaml

```diff
@@ -67,5 +67,12 @@ rules:
   resources:
   - "roles"
   - "rolebindings"
+- verbs:
+  - "*"
```

Comment:
I believe we can get by with just crate here
```suggestion
  - "CREATE"
```

Reply:
Looks like we need Create, Get and List. Without these, the konk operator does not deploy a konk instance.